go.work.example

```diff
@@ -1,27 +0,0 @@
-go 1.20
```

Comment:
can we add this back

Reply:
Must have messed something up with my git. Sorry!